Update files-troubleshoot-smb-connectivity.md (#1316)
MicrosoftDocs/SupportArticles-docs: support/azure/azure-storage/files-troubleshoot-smb-connectivity.md

The resolution to this issue depends on whether this is being caused by an orphaned file handle or lease.

> **Note:** REST leases are used by application to prevent deletion and modification, and the customer should identify which application is acquiring the said leases, before trying to break them, else they could break the application behavior.

Reviewer: ```suggestion
> [!NOTE]
> REST leases are used by applications to prevent files from being deleted or modified. You should identify which application is acquiring the leases before trying to break them, or else you might break the application behavior.
```
